### DISCUSSION TITLE
'BROWSER' not recognized and wait-on code 1 on Windows 10

### DISCUSSION THREAD

**Discussion by TraumatizedTuna** [mifi/lossless-cut#556] [Support] [answered]
Following the instructions provided in developer-notes.md I get the following:
![image](https://user-images.githubusercontent.com/11942607/102467446-97023e00-4050-11eb-850b-3632c8b96471.png)
[2020-12-17T09_02_57_983Z-debug.log](https://github.com/mifi/lossless-cut/files/5708271/2020-12-17T09_02_57_983Z-debug.log)

Upon some research I found that changing "BROWSER=none" to "set BROWSER=none" in package.json seems to fix this problem (although I'll willingly admit that I don't really know what I'm doing, I just wanna get into the JS stuff)
However, this gives me a new error, for which I can't find any useful information at all except that it's possibly some sort of timeout thing:
![image](https://user-images.githubusercontent.com/11942607/102468660-1cd2b900-4052-11eb-9f1e-d93d147f4a74.png)
[2020-12-17T09_05_27_688Z-debug.log](https://github.com/mifi/lossless-cut/files/5708266/2020-12-17T09_05_27_688Z-debug.log)

I'm sorry if this is all my own fault for not knowing exactly what I'm doing, in which case I'd really appreciate to be pointed in the right direction.

**Comment by mifi** [comment]
Ohh the scripts are not updated to work on windows (handles env variables differently). Feel free to submit a pr to fix this. I dont have capacity to test this because i dont run windows

**Reply by TraumatizedTuna** [reply]
Alright, so that first fix would be the proper way to fix it on Windows? Would it screw up Linux compatability?
Unless you have any idea how to fix the second error I guess I'll have to go Linux. Thanks for the reply :)

**Comment by mifi** [comment, marked as answer]
See https://www.npmjs.com/package/cross-env

**Reply by TraumatizedTuna** [reply]
Dude it works! I haven't the faintest idea how it solved my second problem (probably because I haven't the faintest idea what my second problem really was in the first place) but I guess we just made LosslessCut build on Windows. Thank you very much for helping out an underqualified brother in need 🙂
Guess I should do that PR before I start messing with the code
